css/main.css

```diff
@@ -607,6 +607,51 @@ h2 {
   border-top: 1px solid #DDDDDD;
 }
 
+#book-selection-menu-book-list, #book-selection-menu-chapter-list {
+  padding-left: 0.5em;
+  padding-top: 0.5em;
+  padding-bottom: 0.5em;
+}
+
+#book-selection-menu-chapter-list {
+  display: none;
+  padding-left: 1em;
+  padding-right: 1em;
+  width: 33em;
+  min-height: 20em;
+  max-height: 52em;
+  overflow-y: scroll;
```

Comment:
Maybe the extra scroll is a result of this line:
![Screenshot from 2021-06-01 12-50-45](https://user-images.githubusercontent.com/3584119/120374178-d959c780-c2de-11eb-949f-3d2d0de254ca.png)

Reply:
This is necessary because with books like Psalms a scrollbar is needed. Do you see an alternative approach?

Reply:
let me check with `overflow-y: auto`